src/operators/add.ml

```diff
@@ -299,3 +301,4 @@ let () =
          (List.map2 (fun w s -> (w, s)) weights sources)
          video_init video_loop
         :> Source.source))
+*)
```

Comment:
Let's delete dead code or else make it compile.

Reply:
Restored the code :)